### ISSUE TITLE
Confirm transcription direction claim in NucleicSequence

### ISSUE THREAD

**Issue by eblume** [eblume/Tiger-Lily#12]
In the documentation for **tigerlily.sequences.NucleicSequence.translate** I make the following claim:

```
    Conversion is performed by taking each codon and replacing it with
    the corresponding amino acid. In biological terms,
    translate() assumes that the NucleicSequence object's sequence is
    the coding sequence (and not the template). Incidentally, to get
    the coding sequence of a template, just call
    template.reverse_complement().
```
# The Question

Is this true? In other words, given a NucleicSequence, if we claim that that sequence is the template sequence, how can we get the coding sequence? 

Consider: during transcription, the template strand is read in the 3'->5' direction, with the coding sequence being duplicated as mRNA in the corresponding 5'->3' direction.

So the question boils down to this:

**If I give you the sequence 'AGGTAC' and tell you that it is the template sequence, do you have enough information to give me the coding sequence? If I tell you that that template sequence was in the 3'->5' direction, is the corresponding coding sequence reported as the complement of the template or as the reverse complement?**
# The Issue

Based on this answer, update the documentation for **tigerlily.sequences.NucleicSequence.translate** correspondingly.

**Comment by fuzzylogical** [comment]
The main thing to know about the template sequence is that when given a sequence you must know 3 things:
1) the strand is a template or coding strand.
2) the strand's direction, meaning 3'---> 5' or 5' ---> 3' in order to know how to parse the sequence
3) are there any frame shifts that are present that could completely change the result of the mRNA sequence and incidentally the protein will be changed greatly.
Depending on the direction of the sequence it is always the reverse compliment never the compliment of a template strand or coding strand.
tl;dr:
If you are given the coding strand, do a direct translation and you should be fine, if given a template strand do a reverse compliment and translate and you should okay but do remember the possibilities of frameshifts and directional readings (3-->5) and writings(5-->3)

**Comment by fuzzylogical** [comment]
is this an issue that needs to be included in the code or something?

**Comment by eblume** [comment]
It sounds like you are saying that the current documentation is accurate, in which case we can close this ticket. Thanks!
